### ISSUE TITLE
API not working 

### ISSUE THREAD

**Issue by isdzulqor** [ar-android/BelajarMVVM#2]
API not not working. could you give me some sample json data that is needed in this app.

**Comment by ar-android** [comment]
Please signup to ibacor and get the api key and pass your api key as parameter 
example

```java

public interface ApiServices {
    @GET("/api/jadwal-bioskop")
    Observable<ListKota> getListKota(@Query("k")String apiKey);

    @GET("/api/jadwal-bioskop")
    Observable<JadwalBioskop> getjadwalBioskop(@Query("id") int id, @Query("k")String apiKey);
}
```

**Comment by isdzulqor** [comment]
Okey, thank you for your response. but another problem came to me. get list kota is working, but for get detail it got an error 
![image](https://cloud.githubusercontent.com/assets/12388558/23833777/23bd16e6-077d-11e7-911d-68444d20fdbe.png) 
![image](https://cloud.githubusercontent.com/assets/12388558/23833782/2f9bb88c-077d-11e7-9335-aefbd188d501.png)

in which part I can print response data for debugging from request an endpoint?

**Comment by ar-android** [comment]
Yes i make bug for get detail of list kota, and i thing this is good time for you to improve your skill to solve this error.
